tajo-core/src/test/java/org/apache/tajo/querymaster/TestKillQuery.java

```diff
@@ -63,7 +63,7 @@
   private static TajoTestingCluster cluster;
   private static TajoConf conf;
   private static TajoClient client;
-  private static String queryStr = "select t1.l_orderkey, t1.l_partkey, t2.c_custkey " +
+  private static String queryStr = "select t1.l_orderkey, t1.l_partkey, t2.c_custkey, sleep(1) " +
```

Comment:
The problem of a test failure was fixed in https://issues.apache.org/jira/browse/TAJO-1138 and https://issues.apache.org/jira/browse/TAJO-1440.